atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/KeyValueService.java

```diff
@@ -374,15 +375,20 @@ void putUnlessExists(@QueryParam("tableRef") TableReference tableRef,
     void deleteRange(@QueryParam("tableRef") TableReference tableRef, RangeRequest range);
 
     /**
-     * For each cell, deletes all timestamps prior to the associated maximum timestamp, excluding garbage collection
-     * sentinels. Depending on the implementation, this may result in a range tombstone in the underlying KVS.
+     * For each cell, deletes all timestamps prior to the associated maximum timestamp, excluding. Depending on the
```

Comment:
nit: "excluding."

You went a bit far with deleting all "garbage collection sentinels" 😂

Reply:
🎢